src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -48,6 +51,17 @@ type GetQuoteInput = {
   forceQuote?: boolean
 }
 
+type DebouncedQuoteInput = {
+  amount: string
+  sellAsset: Asset
+  buyAsset: Asset
+  feeAsset: Asset
+  action: TradeAmountInputField
+  wallet: HDWallet
+  swapperManager: SwapperManager
+  accountSpecifiersList: Record<string, string>[]
```

Comment:
can we use the source of truth plz such that these don't get out of sync https://github.com/shapeshift/web/blob/develop/src/state/slices/accountSpecifiersSlice/accountSpecifiersSlice.ts#L12